app/routes/surveys/components/SurveyEditor/Question.js

```diff
@@ -116,6 +137,22 @@ const Question = ({ index, question, questionData, deleteQuestion }: Props) => {
           />
         </div>
 
+        <div className={styles.reorderQuestion}>
+          <SelectInput
+            value={{
```

Comment:
You should be able to add `optionComponent={QuestionTypeOption}` and `valueComponent={QuestionTypeValue}` to this `<SelectInput />` in order to style it the same way we do all selects. See https://github.com/webkom/lego-webapp/pull/1771/files#diff-8b2d27bca1f613218ed08ae6d905ae0fR72

You might need to wait for that PR to be merged / steal the code temporarily if you want certain Icons.